spec/00-specification-for-php.md

```diff
@@ -183,8 +183,8 @@ is distributed without any warranty.
     - [The `throw` Statement](11-statements.md#the-throw-statement)
   - [The `try` Statement](11-statements.md#the-try-statement)
   - [The `declare` Statement](11-statements.md#the-declare-statement)
-  - [The `echo` statement](11-statements.md#the-echo-statement)
-  - [The `unset` statement](11-statements.md#the-unset-statement)
+  - [The `echo` Statement](11-statements.md#the-echo-statement)
+  - [The `unset` Statement](11-statements.md#the-unset-statement)
```

Comment:
This is a generated file -- the change should also be in the actual heading named in 11-statements.md.